apps/component-examples/examples/terminal-orders-list.js

```diff
@@ -0,0 +1,89 @@
+require('dotenv').config({ path: '../../.env' });
+
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 3000;
+// const authTokenEndpoint = process.env.AUTH_TOKEN_ENDPOINT;
+// const webComponentTokenEndpoint = process.env.WEB_COMPONENT_TOKEN_ENDPOINT;
+// const subAccountId = process.env.SUB_ACCOUNT_ID;
+const businessId = process.env.BUSINESS_ID;
+// const clientId = process.env.CLIENT_ID;
+// const clientSecret = process.env.CLIENT_SECRET;
+
+app.use(
+  '/scripts',
+  express.static(__dirname + '/../node_modules/@justifi/webcomponents/dist/')
+);
+app.use('/styles', express.static(__dirname + '/../css/'));
+
+// async function getToken() {
+//   const requestBody = JSON.stringify({
+//     client_id: clientId,
+//     client_secret: clientSecret
+//   });
+
+//   let response;
+//   try {
+//     response = await fetch(authTokenEndpoint, {
+//       method: 'POST',
+//       headers: {
+//         'Content-Type': 'application/json',
+//       },
+//       body: requestBody,
+//     });
+//   } catch (error) {
+//     console.log('ERROR:', error);
+//   }
+
+//   const { access_token } = await response.json();
+//   return access_token;
+// }
+
+// async function getWebComponentToken(token) {
+//   const response = await fetch(webComponentTokenEndpoint,
+//     {
+//       method: 'POST',
+//       headers: {
+//         'Content-Type': 'application/json',
+//         Authorization: `Bearer ${token}`,
+//       },
+//       body: JSON.stringify({
+//         resources: [`read:account:${subAccountId}`],
+//       }),
+//     }
+//   );
+
+//   const { access_token } = await response.json();
+//   return access_token;
+// }
+
+app.get('/', async (req, res) => {
+  // const token = await getToken();
+  // const webComponentToken = await getWebComponentToken(token);
+
+  res.send(`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>JustiFi Terminal Orders List Component</title>
+        <script type="module" src="/scripts/webcomponents/webcomponents.esm.js"></script>
+        <link rel="stylesheet" href="/styles/theme.css">
+        <link rel="stylesheet" href="/styles/example.css">
+      </head>
+      <body>
+        <div style="padding:25px;">
```

Comment:
We're probably going to want padding on other components (if we don't already) so maybe it makes sense to create a `component-example-wrapper` class and put it in the `example.css`?

Reply:
When we discussed this in standup this morning I said I would do it in another PR, but I just decided to go ahead and include this in this PR. All the list component example files use this same class now that I added to `example.css`: `.list-wrapper`